app/controllers/criteria_controller.rb

```diff
@@ -128,7 +128,7 @@ def update_positions
 
   def download
     assignment = Assignment.find(params[:assignment_id])
-    criteria = assignment.get_criteria.sort_by(&:position)
+    criteria = assignment.criteria.sort_by(&:position)
```

Comment:
You can remove the `sort_by` here, since the `criteria` association already has an `order` by position.